src/components/FormTitle/FormTitle.scss

```diff
@@ -0,0 +1,16 @@
+@import "./../../globalStyles/variables.scss";
+
+.formTitle {
+    display: flex;
+    background-color: $neutralPaper;
```

Comment:
You dont need to set a background color, its justr a title comp it does not concern itself with what the color behind it is, thats the parents job.